### PR TITLE
Feature/day 05 python

### PR DESCRIPTION
**Pull request by quequitejerina**
- Solution if day 04 exercise

### CODE REVIEW

File: lessons/day_05/python/app.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from ariadne import make_executable_schema, load_schema_from_path
+from ariadne.asgi import GraphQL
+from resolvers import query, skill, person, eye_color
+from resolvers import mutation
```
[the remark this answers is not in the document]
Comment (quequitejerina) [reply lessons/day_05/python/app.py:4]: i was having an line length error. That´s why i did that.

File: lessons/day_05/python/resolvers.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+                )
+                session.execute(add_friend)
+        elif key == "skills":
+            for skill_key in input.get(key):
+                add_skill = person_skills.insert().values(
+                    person_id=person.id, skill_id=skill_key
+                )
+                session.execute(add_skill)
+        else:
+            setattr(person, key, input.get(key))
```
[the remark this answers is not in the document]
Comment (quequitejerina) [reply lessons/day_05/python/resolvers.py:92]: Great!!!

File: lessons/day_05/python/resolvers.py [hunk 3]
```diff
@@ -0,0 +1,127 @@
+from ariadne import QueryType, ObjectType, EnumType, MutationType
+from random import randint
+from models import Skill, Person, person_friends, person_skills
```
Comment (ehx) [inline lessons/day_05/python/resolvers.py:3]: remove unused import 
```suggestion
from models import Skill, Person
```

Comment (quequitejerina) [reply lessons/day_05/python/resolvers.py]: Remember that the IDs it not integer, but a string,

**Pull request merged**